app/scripts/modules/core/help/helpWidget.component.ts

```diff
@@ -0,0 +1,90 @@
+import * as moment from 'moment';
+import {Component, Input, OnChanges, OnDestroy, OnInit, SimpleChanges} from '@angular/core';
+
+interface IHelpFieldContents {
+  content: string;
+  placement: string;
+}
+
+@Component({
+  selector: 'deck-help-widget',
+  template: `
+    <ng-template #popoverContent>
+      <div [innerHTML]="contents.content" (mouseenter)="popoverHovered()" (mouseleave)="hidePopover(false)"></div>
+    </ng-template>
+    <div style="display: inline-block;">
+      <div ng-if="expand && contents.content"
+           class="help-contents small"
+           [popover]="popoverContent"></div>
+      <a href class="help-field" ng-if="!expand && contents.content"
```

Comment:
Do the `ng-if`s not need to be in ng2 syntax? Same with `triggers` and `container` below.